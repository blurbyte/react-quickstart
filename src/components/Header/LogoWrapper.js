import styled from 'styled-components';

const LogoWrapper = styled.header`
  display: inline-flex;
  align-items: center;
`;

export default LogoWrapper;
