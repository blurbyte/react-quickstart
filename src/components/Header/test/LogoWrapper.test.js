import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import LogoWrapper from '../LogoWrapper';

describe('<LogoWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <LogoWrapper />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
