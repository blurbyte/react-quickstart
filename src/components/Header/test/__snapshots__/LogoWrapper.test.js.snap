// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoWrapper /> should render correctly 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<header
  className="c0"
/>
`;
